mdp.ipynb

```diff
@@ -1565,17 +1600,17 @@
     "We cannot have fixed action sequences as the environment is stochastic and we can eventually end up in an undesirable state.\n",
     "Therefore, a solution must specify what the agent shoulddo for _any_ state the agent might reach.\n",
     "<br>\n",
-    "Such a solution is known as a **policy** and is usually denoted by **&#960;**.\n",
+    "Such a solution is known as a **policy** and is usually denoted by $\\pi$.\n",
```

Comment:
I didn't know inline LaTeX was a thing and hence used Unicode. Good job on this one!